src/reference/asciidoc/jdbc.adoc

```diff
@@ -433,6 +433,8 @@ For this purpose, the `INT_CHANNEL_MESSAGE` has a `MESSAGE_PRIORITY` column to s
 In addition, a new `MESSAGE_SEQUENCE` column is also provided to achieve a robust first-in-first-out (FIFO) polling mechanism, even when multiple messages are stored with the same priority in the same millisecond.
 Messages are polled (selected) from the database with `order by MESSAGE_PRIORITY DESC NULLS LAST, CREATED_DATE, MESSAGE_SEQUENCE`.
 
+IMPORTANT: The `capacity` attribute is not allowed when the `message-store` attribute is used.
```

Comment:
I think is correct: just above of that paragraph is located the `Priority Channel` section, this `capacity` restriction applies for PriorityChannel as well. 

Furthermore, just below there is an example showing QueueChannel and PriorityChannel working with together with `JdbcChannelMessageStore` (using `message-store`). So that note applies for both queue and priority.